plugin.audio.ilovemusic/addon.xml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
I did not notice it before, but here is not license here. Please specify the SPDX identifier of your OS license: https://spdx.org/licenses/